README.md

```diff
@@ -2,10 +2,43 @@
 
 The Node.js Foundation Technical Steering Committee is the technical governing body of the Node.js Foundation. It admits and oversees all top-level Projects in the Node.js Foundation. It also elects a representative to the Node.js Foundation Board of Directors.
 
+The TSC is community and contributor driven and is a home for community managed 
+projects and assets. The mission of the Node.js Foundation is to enable and encourage 
+the adoption of Node.js and the TSC is the home of the community driven portion
```

Comment:
I agree that this is poorly worded but I do think we need something that defines the separation.

Sure, the TSC may from time to time get involved in the Marketing Committee or the Board but it is doing so with the mandate that it represents the contributors and the community.